### PR TITLE
[Update] farmOS - fixes

### PR DESCRIPTION
**Pull request by bbiggerr**

### CODE REVIEW

**Comment by nmelehan** [comment]
Deploy preview for *nostalgic-ptolemy-b01ab8* ready!

Built with commit 4eb5e9bf037dd2d11d4fde07b35af91d3fac57c4

https://deploy-preview-3995--nostalgic-ptolemy-b01ab8.netlify.app

File: docs/guides/applications/project-management/install-farmos/index.md [hunk 1]
```diff
@@ -5,137 +5,150 @@ author:
...
 
-5. Finally, after installing the modules, you will be dropped into the FarmOS dashboard:
+1.  Finally, after installing the modules, you are dropped into the farmOS dashboard:
 
     ![welcome](welcome.png)
 
-6. After the installation has finished, you may want to reset your file permissions to avoid security vulnerabilities:
+1.  After the installation has finished, you may want to reset your file permissions to avoid security vulnerabilities from your site's document root. If you installed and configured your Apache server using our [LAMP stack on Debian 10](/docs/guides/how-to-install-a-lamp-stack-on-debian-10/) guide, your document root should be located in the `/var/www/html/example.com/public_html/` directory:
 
        sudo chmod 644 sites/default
```
Comment (andystevensname) [inline docs/guides/applications/project-management/install-farmos/index.md:133]: These lines need one more space for the indent, it looks like.

File: docs/guides/applications/project-management/install-farmos/index.md [hunk 2]
```diff
@@ -5,137 +5,150 @@ author:
...
        sudo chmod 644 ./sites/default/settings.php
 
 ## Add Users
-To add users to your FarmOS distribution, you can do so from the **People** tab under **Manage**.
 
-   ![Add user](Adduser.png)
+To add users to your farmOS distribution, you can do so from the **People** tab under **Manage**.
 
-After each user is created, use the **people** tab, to verify success:
+    ![Add user](Adduser.png)
```
Comment (andystevensname) [inline docs/guides/applications/project-management/install-farmos/index.md:140]: Image does not show up in the guide, might be a spacing issue.

File: docs/guides/applications/project-management/install-farmos/index.md [hunk 3]
```diff
@@ -5,137 +5,150 @@ author:
...
 
-![People screen](peoplescreen.png)
+After each user is created, use the **People** tab to verify success:
+
+    ![People screen](peoplescreen.png)
```
Comment (andystevensname) [inline docs/guides/applications/project-management/install-farmos/index.md:144]: Image does not show up in guide, might be a spacing issue.

**Review by andystevensname — CHANGES_REQUESTED** [review]

**Review by andystevensname — APPROVED** [review]

**Pull request merged**